website/docs/patterns/filter-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,134 @@
+
+**Ex 2.1: Above the data set with filters positioned in a sidebar**
+
+![Applied filters with filter sidebar](/assets/patterns/filter-patterns/applied-filters-positioning-sidebar.png)
+
+If the filters are positioned in a page-level sidebar the layout can be simplified by excluding the applied filters and including an action to clear all of the filters within the sidebar itself.
+
+With this method, the filter values being applied to the dataset are already displayed and implied in the sidebar.
+
+**Ex 2.2: Excludes the applied filters when positioned in a sidebar**
+
+![Applied filters with clear all in the sidebar](/assets/patterns/filter-patterns/applied-filters-positioning-sidebar-clear-all.png)
+
+### Pagination
+
+Use pagination to break down the filtered data set into pages. For more details refer to the [Pagination](/components/pagination) guidelines.
+
+![Pagination example](/assets/patterns/filter-patterns/pagination-layout.png)
+
+## Overflow
+
+Depending on the complexity of the data set or filterable parameters, it may be necessary to account for overflow of elements within the filter bar or in the applied tags.
+
+### Filter bar overflow
+
+If the number of filterable parameters exceeds the available space, consider prioritizing the most commonly used filters in the filter bar and moving less important or more complex filters to a [Flyout](/components/flyout) that can be triggered via an action in the filter bar.
+
+**Ex 3.0: Hiding additional filters behind an action; in this scenario a secondary button**
+
+![More filters trigger](/assets/patterns/filter-patterns/overflow-filter-bar.png)
+
+**Ex 3.1: Triggering a Flyout with additional filter parameters**
+
+![Flyout triggered by more filters](/assets/patterns/filter-patterns/overflow-filter-bar-flyout.png)
+
+!!! Dont
+
+Don't stack multiple filter bars on top of a data set. This can result in unnecessary complexity in the UI.
+
+![Multiple filter bar rows](/assets/patterns/filter-patterns/overflow-filter-bar-multiple-rows.png)
+
+!!!
+
+### Filter sidebar overflow
+
+Similar to the guidelines for the filter bar, an action to view more filters within a [Flyout](/components/flyout) can be triggered within the sidebar orientation.
+
+**Ex 3.2: Hiding additional filters behind an action in a sidebar orientation**
+
+![More filters trigger in a sidebar](/assets/patterns/filter-patterns/overflow-sidebar-more-filters.png)
+
+!!! Info
+
+Consider each one of these scenarios carefully as moving additional filters to an interstitial component like a Flyout adds extra steps in the filtering process that might not be necessary. 
+
+!!!
+
+### Applied filters overflow
+
+If the number of filterable parameters is large, there's a likelihood that the number of applied filters may also extend to multiple lines. In this scenario the same methods apply as the filter bar and sidebar.
```

Comment:
> In this scenario the same methods apply as the filter bar and sidebar.

Have you thought about something more like a show/hide that expands the FilterList instead of opening a Flyout to show the other filters? As you noted, these are here to help with easy and quick dismissal, so why would we hide them within a Flyout?

Reply:
My intention here (and with the overflow content in general) was to only use components/patterns/things are the currently supported within Helios. We briefly discussed whether we should recommend a reveal/show-hide based on the work in progress, but since it isn't complete yet decided against.